vpdq/cpp/hashing/filehasher.cpp

```diff
@@ -386,38 +523,39 @@ bool hashVideoFile(
     // https://github.com/FFmpeg/FFmpeg/blob/6a9d3f46c7fc661b86192e922ab932495d27f953/doc/examples/decode_video.c#L182
 
     try {
-      ret = processFrame(
-          packet,
-          frame,
-          targetFrame,
-          swsContext,
-          codecContext,
-          phasher,
-          pdqHashes,
-          frameRate,
-          verbose,
-          frameNumber,
-          frameMod);
+      hasher.processPacket(std::move(packet));
     } catch (const std::runtime_error& e) {
       std::cerr << "Flushing frame buffer failed: " << e.what() << std::endl;
       failed = true;
     }
-
-    av_packet_unref(packet);
   }
 
-  av_packet_free(&packet);
-  sws_freeContext(swsContext);
-  av_freep(targetFrame->data);
-  av_frame_free(&targetFrame);
-  av_frame_free(&frame);
-  avcodec_free_context(&codecContext);
-  avformat_close_input(&formatContext);
+  if (thread_count != 1) {
```

Comment:
nit, ignorable: In some cases you use the multithreaded variable, in some cases you use this check. Seems like consistently using this  version of the check might be less error prone than using the separate `multithreaded` variable.